modules/file-system/parallelstore/variables.tf

```diff
@@ -24,6 +24,20 @@ variable "deployment_name" {
   type        = string
 }
 
+variable "daos_agent_config" {
```

Comment:
I am not sure if adding new variables would be a better choice. Since the underlying thought process behind toolkit is, how you can use same interface for all file systems, like I just need to change 1 line to deploy parallelstore from filestore in blueprint.

Reply:
Frankly, I see no problem here. Added variables are parallelstore specific and go into parallelstore module. @harshthakkar01 what would be a better alternative?

Reply:
Had an offline discussion and we agreed on moving on with this approach.